### ISSUE TITLE
Put WindowsDesktop shared framework bundle installer through loc process

### ISSUE THREAD

**Issue by dagood** [dotnet/windowsdesktop#921]
There are new strings with the WindowsDesktop shared framework bundle installer (at minimum `.NET Core Runtime` => `Windows Desktop Runtime`) that need to be localized.

Currently blocked on getting the branding info to localize: https://github.com/dotnet/core-setup/issues/7530.

**Comment by RussKie** [comment]
@dagood is this still a thing?

**Comment by dagood** [comment]
This is a preemptively filed followup issue to https://github.com/dotnet/windowsdesktop/issues/191.

/cc @NikolaMilosavljevic

**Comment by RussKie** [comment]
#191 was resolved by https://github.com/dotnet/windowsdesktop/pull/769, which introduced loc'ed strings. 
Presume fixed.
